src/modules/auth/actions.js

```diff
@@ -39,6 +40,12 @@ export const fetchServiceDependentData = () => dispatch => Promise.all([
     fetchExtraPropertiesSchemaTypes,
 ].map(a => dispatch(a())));
 
+const getDatasetsByID = (state) => Object.fromEntries(
```

Comment:
this should probably be a generic helper fuynction in another module

Reply:
I am trying to figure out where to place it... Could we leave it in here and then see where it belongs?